LanguageFeatures/Extension-types/nullability_A01_t01.dart

```diff
@@ -2,13 +2,11 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion Let V be an extension type of the form Name<T1, .. Ts>, and let R
-/// be the corresponding instantiated representation type. If R is non-nullable
-/// then V is a proper subtype of Object, and V is non-nullable. Otherwise, V is
-/// a proper subtype of Object?, and V is potentially nullable.
+/// @assertion An extension type `V` is a proper subtype of `Object?`. It is
+/// potentially non-nullable, unless it implements `Object` or a subtype thereof
 ///
-/// @description Checks that an extension type is not-nullable even if its
-/// representation type is nullable
+/// @description Checks that an extension type is not-nullable even if doesn't
+/// implement subtype of `Object` and its representation type is nullable
```

Comment:
`extension type E(R it) implements Object {}` is always an error in the case where `R` is nullable, so we shouldn't actually need the added words.

Maybe express the same thing as 'commentary', if that concept exists here? Something like this could be added at the end of this paragraph: 'In this case we know that the extension type doesn't implement `Object` or a subtype thereof, because a nullable representation type would then be a compile-time error'.

Reply:
Added `nullability_A01_t04.dart` testing that it is a compile-time error if an extension type does implement subtype of `Object` and its representation type is nullable.

Reply:
OK, thanks!